### ISSUE TITLE
Cannot Patch Odyssey into sm64 us wii vc wad

### ISSUE THREAD

**Issue by mcgrog** [Admentus64/Patcher64Plus-Tool#1]
steps taken:
1) selecting a sm64 us wii vc wad that patch options section appears to work with - (got black bar removal to work, 60fps is questionable)
2) selecting working odyssey rom 65536kb
3) hitting inject
4) attempting to play through dolphin before running on wii

alternate steps taken:
1) select sm64 us wii vc wad that works
2) browse to odyssey patch included in files
3) patch rom
4) attempting to play through dolphin before running on wii

ive tried injecting then patching using included file as well but to no avail, if this script can patch odyssey perhaps include a more detailed or even more simplified "how" in the readme, have tried a fair whack of internet digging and this seems like the best way to get sm64 odyssey running on my wii, love the effort on the 3k lines of powershell by the way, youve done an epic job wrapping these tools together into winforms

**Comment by Admentus64** [comment]
Nope. Sadly that won't work. This is an issue with the how Nintendo wrote their own Virtual Console emulators. In other words the Virtual Console emulator provided with Super Mario 64 is not accurate enough to run the ROM hack Super Mario Odyssey.

All ROM hacks which are included with the Patcher64+ Tool that are only available in Nintendo 64 mode have also been tested in Wii VC mode, and those who were not compatible in Wii VC mode have been removed from the Wii VC mode GUI.

Yes, there are a lot of Super Mario 64 ROM hack which are not compatible in Wii VC mode. Another ROM hack that fails on the Wii VC is Super Smash Bros Redux.

I am not sure what specifically causes these ROM hacks to fail, but there are to key factors here:
- Usage of the Expansion Pak (for a total of 8MB)
- ROM size

In other words, a ROM Hack has to follow the same specifications as the original game for which the VC emulator was adjusted for. Meaning for Super Mario 64 that would be the regular 4MB RAM Memory and an 8MB ROM file size.

So what about the Star Road ROM Hack working on the Wii VC? Well... I kinda replaced the whole VC emulator for it to get it working. Don't ask me how. I didn't make the replacement VC emulator for it. So in order to get ROM hacks like Super Mario Odyssey working we would need a replacement for the VC emulator.

At least that is my extent of knowledge to it so far.
